### PR TITLE
GitHub login

### PR DESCRIPTION
**Pull request by mateuszpalyz**
Add ability to login using github account, update some icons.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -127,6 +127,16 @@ def twitter
     get_authorization(:twitter)
   end
 
+  # Returns true when this user has a twitter authorization
+  def github?
+    has_authorization?(:github)
+  end
+
+  # Returns the twitter authorization
```
Comment (colindean) [inline app/models/user.rb]: doc not in sync with method. maybe call these methods `github_authorization` and `github_authorization?`

**Comment by colindean** [comment]
Pretty cool. I'll let one of the other committers take a look and merge.

**Comment by mateuszpalyz** [comment]
@colindean I'll fix docs, what do you think about adding login via facebook?

**Comment by colindean** [comment]
Facebook login existed at one point. We ripped it out for reasons I've long forgotten, but seemingly related to one of the project founders deleting their Facebook account and it disabled our production key.

**Comment by colindean** [comment]
Closing to clear this from my queue. Sadly, rstat.us development halted a while ago.

**Pull request closed without merging**